transport.go

```diff
@@ -21,15 +21,17 @@ func ConnectionFailure(*http.Request) (*http.Response, error) {
 
 // NewMockTransport creates a new *MockTransport with no responders.
 func NewMockTransport() *MockTransport {
-	return &MockTransport{make(map[string]Responder), nil}
+	return &MockTransport{make(map[string]Responder), nil, make(map[string]int), 0}
 }
 
 // MockTransport implements http.RoundTripper, which fulfills single http requests issued by
 // an http.Client.  This implementation doesn't actually make the call, instead deferring to
 // the registered list of responders.
 type MockTransport struct {
-	responders  map[string]Responder
-	noResponder Responder
+	responders     map[string]Responder
+	noResponder    Responder
+	callCountInfo  map[string]int
+	totalCallCount int
```

Comment:
No Worries!
Yes just keeping an eye on it but not doing anything for now because of the race conditions. I've written my own test helpers.